Client/Frontend/Settings/SearchBar/SearchBarSettingsViewModel.swift

```diff
@@ -79,16 +76,16 @@ private extension SearchBarSettingsViewModel {
 
     /// New user defaults to bottom search bar, existing users keep their existing search bar position
```

Comment:
We can adjust the comment here for this function I think
`New user defaults to bottom search bar` isn't correct anymore in all cases